src/select.coffee

```diff
@@ -409,9 +415,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
       value = content.objectAt index
     else # setter
       index = content.indexOf value
-      @setHighlightedIndex index, yes
+      @setHighlightedIndex index, @get('shouldEnsureVisible')
```

Comment:
Btw, when there is only one item, why shouldn't it jump to that highlighted item? After this change, it won't, right?